.travis.yml

```diff
@@ -165,6 +165,46 @@ jobs:
       script: 'for i in $(seq 0 4); do printf "[XXX: Run #%s]\n" "$i";docker run -it -v "${PWD}:/netdata:rw" -w /netdata "fedora:31" tests/updater_checks.sh && break; done'
       after_failure: post_message "TRAVIS_MESSAGE" "Netdata updater process failed on bare Fedora 31"
 
+    - name: DEB package test
```

Comment:
In fact just move all this to a GHA workflow 👍

Reply:
I intentionally put it in Travis because that's the environment where we're building these packages, so we can be sure that we've got the exact same environment.